hax/treemakers/posrec.py

```diff
@@ -90,7 +91,8 @@ def __init__(self):
 
     def load_nn(self):
         """For loading NN files"""
-
+        from keras.models import model_from_json
```

Comment:
It's really sloppy to move imports into packages since machines are setup wrong.  Can you instead look at Mocking, which was the solution I suggested when people were asking about this on Gitter?

Reply:
They are just following our illustrious example at https://github.com/XENON1T/pax/blob/9ecd5c0b94a295893f0bc4eeb0be31039019cd9c/pax/formats.py#L174 :-)

Keep in mind hax.init()  hits all the global imports in treemaker files (since it must figure out what treemakers exist). I don't really want to pull in all the random libraries and files people are using in treemakers -- even if I do have them. I've already been cursing at the corrections treemaker for loading correction maps at import time, now I have to keep pax up to date to even use hax... 

On the other hand if everybody moves their imports inwards dependency tracking becomes a problem. Not sure what wisdom is.

Reply:
@tunnell This issue is slowing down some analysts on Midway and OSG. Both admins are aware of this, but probably will take a while to get a fix from their side. As @JelleAalbers there are probably a few places already where this would need to be improved (or maybe it's actually the desired method?). Not sure if @jpienaar13 can quickly learn about Mocking, so can we merge this for now?